lib/validators/options_processor/teams.js

```diff
@@ -47,4 +67,35 @@ const getTeamMembers = async (context, team) => {
   return res.data.map(member => member.login)
 }
 
+const getTeamMembershipState = async (context, team, username) => {
+  const stringArray = team.split('/')
+  if (stringArray.length !== 2) {
+    throw Error(`each team id needs to be given in 'org/team_slug'`)
+  }
+  if (stringArray[0].indexOf('@') === 0) stringArray[0] = stringArray[0].substring(1)
+
+  const org = stringArray[0]
+  const teamSlug = stringArray[1]
+
+  let res
+  try {
+    res = await context.github.teams.getMembershipInOrg({
+      org,
+      team_slug: teamSlug,
+      username: username
+    })
+  } catch (err) {
+    if (err.status === 404) {
+      return false
+    }
+    throw err
+  }
+
+  if (res.data.state === 'active') {
```

Comment:
You can replace L94 - 98 in a succinct manner:

` return (res.data.state === 'active')`